packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -379,8 +379,6 @@ export default class ComboBox extends React.Component {
                   id,
                   placeholder,
                   onKeyDown: event => {
-                    event.stopPropagation();
-
                     if (match(event, keys.Enter)) {
```

Comment:
I _think_ this will end up breaking for folks trying to enter in a field with a space (it will toggle the menu)

Should we do something like:

```js
if (matches(event, [keys.Enter, keys.Space])) {
  event.stopPropagation();
  if (match(event, keys.Enter)) {
    toggleMenu();
  }
}
```

Reply:
yes. like we discussed, I added the check for space key to `stopPropagation`